public/style.css

```diff
@@ -71,22 +60,33 @@ form > * {
     padding-top: 2rem;
 }
 
-#qrcode {
-    margin: 2rem;
-}
+
 
 input {
     text-align: right;
 }
 
+legend {
+    color: var(--secondary);
+}
+
+#qrcode {
+    margin: 2rem;
+}
+
 .invisible {
     display: none;
 }
 
+.colored-button {
+    color: black;
+    background: var(--secondary);
+}
+
```

Comment:
You missed the checkbox by styling this with a custom class. When you inspect the buttons you applied this too, you can see they're already styled with `--buttonColor` and `--buttonBg`, which is the place for this change.

As-is this change is a regression